app/cells/plugins/core/cell.rb

```diff
@@ -2,6 +2,10 @@ module Plugins
   module Core
     class Cell < FieldCell
       view_paths << "#{Cortex::Plugins::Core::Engine.root}/app/cells"
+
+      def required?
+        field_item.field.validations["presence"] == true
```

Comment:
I know we've used "presence" for a while but why name the validation "presence" and not "required"?